docs/src/whatsnew/latest.rst

```diff
@@ -27,7 +27,8 @@ This document explains the changes made to Iris for this release
 ================
 
 #. Welcome to `@wjbenfold`_, `@tinyendian`_, `@larsbarring`_, `@akuhnregnier`_
```

Comment:
```suggestion
#. Welcome to `@wjbenfold`_, `@tinyendian`_, `@larsbarring`_, `@akuhnregnier`_,
```

Reply:
D'oh! Forgot about the comma! Thanks for catching this  @rcomer !